src/plugins/crypto/README.md

```diff
@@ -126,6 +126,16 @@ The following example configuration illustrates this concept:
 
 Only keys marked with a certain meta-key will be considered for encryption/decryption.
 
+## Configuration ##
+
+The following key must be set to `"true"` within the plugin configuration,
+if the plugin should shut down the crypto library:
+
+	/crypto/shutdown
```

Comment:
Why use prefix for configuration? If you want it plugin-specific, you can pass it only to plugin. And if you want it for the backend, you pass it as backend config.